src/fides/api/ops/models/messaging.py

```diff
@@ -12,32 +12,52 @@
     StringEncryptedType,
 )
 
-from fides.api.ops.common_exceptions import EmailDispatchException
+from fides.api.ops.common_exceptions import MessageDispatchException
 from fides.api.ops.db.base_class import JSONTypeOverride
-from fides.api.ops.schemas.email.email import (
-    SUPPORTED_EMAIL_SERVICE_SECRETS,
-    EmailServiceSecretsMailgun,
-    EmailServiceType,
+from fides.api.ops.schemas.messaging.messaging import (
+    EMAIL_MESSAGING_SERVICES,
+    SMS_MESSAGING_SERVICES,
+    SUPPORTED_MESSAGING_SERVICE_SECRETS,
+    MessagingMethod,
+    MessagingServiceSecretsMailgun,
+    MessagingServiceSecretsTwilioEmail,
+    MessagingServiceSecretsTwilioSMS,
+    MessagingServiceType,
+)
+from fides.api.ops.schemas.messaging.messaging_secrets_docs_only import (
+    possible_messaging_secrets,
 )
-from fides.api.ops.schemas.email.email_secrets_docs_only import possible_email_secrets
 from fides.api.ops.util.logger import Pii
 from fides.ctl.core.config import get_config
 
 CONFIG = get_config()
 logger = logging.getLogger(__name__)
 
 
+def get_messaging_method(
+    service_type: Optional[str],
```

Comment:
Right now I'm handling this case in the `message_dispatch_service.py`. But now that I think of it, I'm implicitly erroring if the configured `service_type` does not match any of our accepted messaging service types (email, sms). I should instead explicitly check for `None`.

My inclination is surfacing this error within the `message_dispatch_service.py` instead of in the model, since we may want to call `get_messaging_method` from another place in the code (e.g. a new endpoint) where it won't make sense to throw an err if a messaging method hasn't been configured.